tests/slr_integration/Editor_Test.php

```diff
@@ -230,4 +231,83 @@ public function test_get_store_data( \Closure $fixture ): void {
 		);
 		$this->assertMatchesJsonSnapshot( $json );
 	}
-}
\ No newline at end of file
+
+	public function test_it_should_update_slr_flags_on_ticket_save() {
+		$id = tribe_events()->set_args( [
+			'title'      => 'Test Event',
+			'start_date' => '+1 week',
+			'duration'   => 3 * HOUR_IN_SECONDS,
+		] )->create()->ID;
+
+		$ticket_1 = $this->create_tc_ticket( $id, 10.10 );
+
+		$this->assertFalse( (bool) get_post_meta( $id, Meta::META_KEY_ENABLED, true ) );
+		$this->assertEmpty( get_post_meta( $id, Meta::META_KEY_LAYOUT_ID, true ) );
+		$this->assertFalse( (bool) get_post_meta( $ticket_1, Meta::META_KEY_ENABLED, true ) );
+		$this->assertEmpty( get_post_meta( $ticket_1, Meta::META_KEY_SEAT_TYPE, true ) );
+
+		$ticket_data = [
+			'tribe-ticket' => [
+				'seating' => [
+					'enabled'  => 1,
+					'seatType' => 'seat-type-uuid-1',
+					'layoutId' => 'layout-uuid-1',
+					]
+			],
+		];
+
+		$this->make_controller()->register();
+
+		do_action( 'tribe_tickets_ticket_added', $id, $ticket_1, $ticket_data );
+
+		$this->assertTrue( (bool) get_post_meta( $id, Meta::META_KEY_ENABLED, true ) );
+		$this->assertEquals( 'layout-uuid-1', get_post_meta( $id, Meta::META_KEY_LAYOUT_ID, true ) );
+		$this->assertTrue( (bool) get_post_meta( $ticket_1, Meta::META_KEY_ENABLED, true ) );
+		$this->assertEquals( 'seat-type-uuid-1', get_post_meta( $ticket_1, Meta::META_KEY_SEAT_TYPE, true ) );
+	}
+
+	public function test_it_should_skip_capacity_storage_when_revision() {
+		$event_id = tribe_events()->set_args(
+			[
+				'title'      => 'Test Event',
+				'status'     => 'publish',
+				'start_date' => '2020-01-01 12:00:00',
+				'duration'   => 2 * HOUR_IN_SECONDS,
+			]
+		)->create()->ID;
+
+		// Enable the global stock on the Event.
+		update_post_meta( $event_id, Global_Stock::GLOBAL_STOCK_ENABLED, 1 );
+
+		// Set the Event global stock level to 100.
+		update_post_meta( $event_id, Global_Stock::GLOBAL_STOCK_LEVEL, 100 );
+
+		update_post_meta( $event_id, Meta::META_KEY_ENABLED, true );
+		update_post_meta( $event_id, Meta::META_KEY_LAYOUT_ID, 1 );
+
+		$ticket_id = $this->create_tc_ticket(
+			$event_id,
+			10,
+			[
+				'tribe-ticket' => [
+					'mode'     => Global_Stock::CAPPED_STOCK_MODE,
+					'capacity' => 30,
+				],
+			]
+		);
+
+		update_post_meta( $ticket_id, Meta::META_KEY_ENABLED, true );
+		update_post_meta( $ticket_id, Meta::META_KEY_SEAT_TYPE, 'seat-type-uuid-A' );
+
+		// $revision_id = wp_save_post_revision( $event_id );
```

Comment:
Remove this left-over debug code?

Reply:
Removed the comment, its covering the revision case